PHASE_1/API_SourceCode/src/api/services/Report.service.ts

```diff
@@ -23,4 +24,26 @@ export class ReportService {
     };
     return result;
   }
+
+  async getSpecificReport(
+    reportId: string
+  ): Promise<IReportSpecificSuccessResponse> {
+    const report: ReportEntity = await this.reportRepository.getSpecificReport(
+      reportId
+    );
+
+    if (report === undefined) {
+      this.logger.error(`No report found in db`);
+      throw new HTTPError({
```

Comment:
this is fine, can we add this as a new error in Constants.ts ?
then we can just import it and use here